sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -10,13 +10,16 @@
 - Removed `get_form_training_client` from `FormRecognizerClient`
 - Added `get_form_recognizer_client` to `FormTrainingClient`
 - A `HttpResponseError` is now raised if a model with `status=="invalid"` is returned from the `begin_train_model()` or `train_model()` methods
+- `PageRange` is renamed to `FormPageRange`
+- `FormField` does not have a page_number.
+- `begin_recognize_receipts` APIs now return `RecognizedReceipt` instead of `USReceipt`
+- `USReceiptType` is renamed to `ReceiptType`
```

Comment:
I think we should include that we added `RecognizedReceipt` class, that it inherits from `RecognziedForm` and `USReceipt` inherits from it.

Reply:
RecognizedReceipt is more of a "middle man" and is mainly for documenting purposes - it should not ( and does not) change the behaviour of UsReceipt - so i  dont think it should be in changelog